driver/pace/driver/diagnostics.py

```diff
@@ -34,6 +36,35 @@ def cleanup(self):
         ...
 
 
+@dataclasses.dataclass
+class ZSelect:
+    level: int
```

Comment:
what is the main motivation behind 1 level for all names?

Reply:
That's the more intuitive way for me. Also, ZSelect implies that we are selecting by z level. So I think this makes sense.